R/DataMart.R

```diff
@@ -346,6 +346,40 @@ QSaveData <- function(object, filename, compression.file.size.threshold = NULL,
     }
 }
 
+#' Deletes a set of objects
+#'
+#' Deletes a list of objects by filename from the Displayr cloud drive
+#'
+#' @param filenames collection of character strings. Names of the files to delete.
+#'
+#' @importFrom httr DELETE add_headers
+#' @importFrom utils URLencode
+#' 
+#' @return NULL invisibly. Called for the purpose of deleting data
+#' and assumed to succeed if no errors are thrown.
+#' 
+#' @export
+QDeleteFiles <- function(filenames, company.token = NA)
+{
+    company.secret <- if (missing(company.token)) getCompanySecret() else company.token
```

Comment:
The default of `NA` will mean `missing` never returns `TRUE`. The `missing` function checks if the argument isnt explicitly called and has no default. I'd recommend updating the signature to `company.secret)`